### PR TITLE
Add merge state protobuf 

### PR DESCRIPTION
**Pull request by terencechain**
This PR adds merge version beacon state protobuf definition

For reviewers, please 🙏 
 - Cross check definitions defined in [spec](https://github.com/ethereum/consensus-specs/blob/dev/specs/merge/beacon-chain.md#executionpayloadheader)

### CODE REVIEW

File: proto/prysm/v1alpha1/beacon_state.proto [hunk 1]
```diff
@@ -186,3 +186,68 @@ message SyncAggregatorSelectionData {
...
+  Checkpoint current_justified_checkpoint = 8003;
+  Checkpoint finalized_checkpoint = 8004;
+
+  // Altair fields [9001-10000]
+  repeated uint64 inactivity_scores = 9001 [(ethereum.eth.ext.ssz_max) = "1099511627776"];
+  SyncCommittee current_sync_committee = 9002; // [New in Altair]
+  SyncCommittee next_sync_committee = 9003; // [New in Altair]
+
+  // Merge fields
+  ExecutionPayloadHeader latest_execution_payload_header = 10000; // [New in Merge]
```
Comment (rkapka) [inline proto/prysm/v1alpha1/beacon_state.proto:235]:
```suggestion
  ExecutionPayloadHeader latest_execution_payload_header = 10001; // [New in Merge]
```

File: proto/prysm/v1alpha1/beacon_state.proto [hunk 2]
```diff
@@ -186,3 +186,68 @@ message SyncAggregatorSelectionData {
...
+  Checkpoint previous_justified_checkpoint = 8002;
+  Checkpoint current_justified_checkpoint = 8003;
+  Checkpoint finalized_checkpoint = 8004;
+
+  // Altair fields [9001-10000]
+  repeated uint64 inactivity_scores = 9001 [(ethereum.eth.ext.ssz_max) = "1099511627776"];
+  SyncCommittee current_sync_committee = 9002; // [New in Altair]
+  SyncCommittee next_sync_committee = 9003; // [New in Altair]
+
+  // Merge fields
```
Comment (rkapka) [inline proto/prysm/v1alpha1/beacon_state.proto:234]:
```suggestion
  // Merge fields [10001-11000]
```

**Review by rkapka — COMMENTED** [review]
What about `ExecutionPayload`?

**Comment by rauljordan** [comment]
Few comments from @rkapka @terencechain

**Comment by terencechain** [comment]
@rkapka @rauljordan ready again, thanks!

**Review by rauljordan — APPROVED** [review]

**Pull request merged**